### PR TITLE
[new release] mrmime (0.1.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Mr. MIME

- Project page: <a href="https://github.com/mirage/mrmime">https://github.com/mirage/mrmime</a>
- Documentation: <a href="https://mirage.github.io/mrmime/">https://mirage.github.io/mrmime/</a>

##### CHANGES:

- First release

### CODE REVIEW

File: packages/mrmime/mrmime.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
+
+pin-depends: [
+  [ "crowbar.dev" "git+https://github.com/stedolan/crowbar.git" ]
+]
+
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency anymore (see #14266)

File: packages/mrmime/mrmime.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name:         "mrmime"
+version:      "dev"
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]:
```suggestion
```
This is going to force the version to `dev` and not `0.1.0`. Also the field is unnecessary.

File: packages/mrmime/mrmime.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/mirage/mrmime"
+bug-reports:  "https://github.com/mirage/mrmime/issues"
+dev-repo:     "git+https://github.com/mirage/mrmime.git"
+doc:          "https://mirage.github.io/mrmime/"
+license:      "MIT"
+synopsis:     "Mr. MIME"
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]: Could we have something a bit more elaborate?

File: packages/mrmime/mrmime.0.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+description:  """Parser and generator of mail in OCaml"""
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]: I would more see this in synopsis instead and have either an even more elaborate description or just remove the description field.

File: packages/mrmime/mrmime.0.1.0/opam [hunk 5]
```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name:         "mrmime"
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]: This field is also unnecessary

File: packages/mrmime/mrmime.0.1.0/opam (hunk 1 above)
Comment (dinosaure) [reply packages/mrmime/mrmime.0.1.0/opam]: Oh ok I did not know!

File: packages/mrmime/mrmime.0.1.0/opam [hunk 6]
```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/mirage/mrmime"
+bug-reports:  "https://github.com/mirage/mrmime/issues"
+dev-repo:     "git+https://github.com/mirage/mrmime.git"
+doc:          "https://mirage.github.io/mrmime/"
+license:      "MIT"
+synopsis:     "Parser and generator of mail in OCaml"
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]:
```suggestion
synopsis:     "Email parser and generator in OCaml"
```

File: packages/mrmime/mrmime.0.1.0/opam [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+description:  """Parser and generator of mail in OCaml according RFC822, RFC2822 and RFC5322"""
```
Comment (kit-ty-kate) [inline packages/mrmime/mrmime.0.1.0/opam]:
```suggestion
description:  """Email parser and generator in OCaml according RFC822, RFC2822 and RFC5322"""
```

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**